src/avt/Pipeline/Data/vtkmDataSet.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
```

Comment:
should we call this `visitVtkmDataset`, or `vtkmDataSetWrapper` -- something to reflect its a wrapper around the native dataset?

Reply:
How about `avtVtkmDataSet`?

Reply:
`avtVtkmDataSet` sounds good!